USAGE.md

```diff
@@ -1,24 +1,25 @@
 # Using the Geospatial Data Lake
 
-The purpose of GDL is to store geospatial datasets. This document should provide the technical know-how to create and maintain such datasets.
+The purpose of GDL is to store geospatial datasets. This document should provide the technical
+know-how to create and maintain such datasets.
 
-The keywords "must", "must not", "required", "shall", "shall not", "should", "should not", "recommended",  "may", and "optional" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+The keywords "must", "must not", "required", "shall", "shall not", "should", "should not",
+"recommended",  "may", and "optional" in this document are to be interpreted as described in [RFC
+2119](https://tools.ietf.org/html/rfc2119).
 
-## Prerequisites
-Currently, Data Lake allows read/write access for all users authenticated via `DATALAKE_SAML_IDENTITY_PROVIDER_ARN` during deployment time (see [README](README.md#aws-infrastructure-deployment-cdk-stack)).
-
-The product team will provide you the following information to enable to you to start using the Data Lake:
-
-Also, following information must be provided by Data Lake instance maintainer in order to star using it:
+# Prerequisites
+## Data Lake account and resource names
+Following information must be provided by Data Lake instance maintainer in order to start using it:
 
 - Data Lake AWS account ID (`DATALAKE_AWS_ACCOUNT_ID`)
 - Data Lake user role name (`DATALAKE_USER_ROLE_NAME`)
 - Data Lake lambda function endpoint names (`DATALAKE_LAMBDA_FUNCTION_ENDPOINT_NAME`)
 
-### Data Maintainers
-To import data in to the Data Lake, you will need a 'staging' S3 bucket with your data in it. You will need to give permissions to Data Lake AWS account to read your data.
+## Staging bucket permissions
+To import data in to the Data Lake, Staging S3 Bucket (dataset source bucket) must be readable by
+Data Lake.
 
-Example bucket policy:
+Example Staging Bucket policy:
```

Comment:
We have a *specific* bucket called the "staging bucket", but this configuration would work with any source bucket. Would it make sense to call it the "source bucket" here?

Reply:
Changed to "dataset source bucket"